.github/workflows/zizmor.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       security-events: write
     steps:
       - name: Checkout repository
-        uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5.0.0 + a doc-only README update
+        uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
```

Comment:
I have confirmed that 1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 corresponds to v6.0.0. I was very confused because https://github.com/actions/checkout/releases/tag/v6 now points at https://github.com/actions/checkout/commit/8e8c483db84b4bee98b60c0593521ed34d9990e8, which is 6.0.1, but https://github.com/actions/checkout/commits/v6.0.0/ has the correct history with 1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 at its tip. Very confusing.